examples/dataservice-example/ProtectedCacheExample.cpp

```diff
@@ -0,0 +1,104 @@
+/*
```

Comment:
Why do we need to test DefaultCache raw?
The idea is that we test protected/mutable cache together with VersionedLayerClient::GetData()

Reply:
changed